geetools/Image.py

```diff
@@ -1427,3 +1427,18 @@ def plot(
             gdf = gpd.GeoDataFrame.from_features(fc.getInfo()["features"])
             gdf = gdf.set_crs("EPSG:4326").to_crs(crs)
             gdf.boundary.plot(ax=ax, color=color)
+
+    @classmethod
+    def fromList(cls, image_list: ee_list):
+        """Create a single image by passing a list of images.
+
+        Parameters:
+            image_list: a list of ee.Image
+
+        Returns:
+            A single ee.Image with one band per image in the passed list
+        """
+        ilist = ee.List(image_list)
+        i0 = ee.Image(ilist.get(0))
+        rest = ee.List(ilist.slice(1))
+        return i0.geetools.merge(rest)
```

Comment:
I'm in general against reusing geetools methods within other classes. Not because it makes no sense but more because I don't know the import order of the different geetools extentions. in this case when you call ee.image extention from the extention itself on an object that is not `self._obj`, I really don't know what is making it work. The import statement in the tests ? The import order in \ __init__.py` ? dark magic ? 

As I don't really know what if instead you use a vanilla GEE shortcut as: 
```python 
return ee.ImageCollection(images).toBands()
```

Reply:
Haha, ok, I have solved it in this way several times so it could work. I will need to rename the bands 'cos `toBands()` creates new names. This makes me think I haven't tried using two images with the same band name in the current implementation.

Reply:
There is an issue when band names are repeated that needs to be resolved, and thus a decision needs to be taken:
1. we don't allow repeated band names

In this case we could change the approach (as you suggest) and do

``` python
bandNames = images.map(lambda i: ee.Image(i).bandNames()).flatten()
image = ee.ImageCollection.fromImages(images).toBands().rename(bandNames)
```
And if a band name is repeated it will raise an error saying something like

> ee.ee_exception.EEException: Image.rename: Can't add a band named 'a' to image because a band with this name already exists. Existing bands: [a, b, c].

2. we allow repeated band names and handle the renaming in our side

In this case we could also change the approach, but the renaming function also needs to be discussed

``` python
bandNames = images.map(lambda i: ee.Image(i).bandNames()).flatten()
## handle duplicated names here
image = ee.ImageCollection.fromImages(images).toBands().rename(bandNames)
```

3. we allow repeated band names and use GEE renaming convention

In this case we can leave the approach as it is now, and the renaming will be handled by the method `.addBands` used in `ee.Image.geetools.merge`.

IMHO, if we cannot reuse `geetools`'s methods in other `geetools` methods, it's a big limitation. Can't we just trust the dark magic? 🙈​🙉​🙊​

Reply:
I checked the exact `to_bands()` behaviour and whatever we do it still build weird (but always working) band names: 
`<image_Id>_<band_name>`. So I think we can reasonably reduce a bit the scope and focus on multiple images with multiple bands but all with different bandNames which is option 1. That should cover 90% of the use cases and for the other , users can go back to the vanilla implementation.

Reply:
Ok, since we cannot customize the error, we can tell the user (in the docstring) that if there are duplicated bands the error will look like that. BTW, I didn't get what you meant by _"use the vanilla implementation"_, which? `ee.Image([i1,in])` ?? that won't work... anyways, I agree that it's an edge case and probably it'll never happen, and if it does, I hope that user is smart enough to, at least, create an issue in this repo 😆

Reply:
+ 1 for the docstring notification
what I meant by vanilla is write down the  `ee.IMageCollection(lis_image).toBands()` and deal with the band names as they see fit.